prime-router/settings/organizations.yml

```diff
@@ -987,4 +987,28 @@
         host: sftp
         port: 22
         filePath: ./upload
-        credentialName: DEFAULT-SFTP
\ No newline at end of file
+        credentialName: DEFAULT-SFTP
+- name: "CDC-ELIMS-HL7"
+  description: "Centers for Disease Control ELIMS System"
+  jurisdiction: "FEDERAL"
+  stateCode: null
+  countyName: null
+  filters: []
+  featureFlags: []
+  keys: []
+  senders:
+  - !<full-elr>
+    name: "elr"
+    organizationName: "CDC-ELIMS-HL7"
+    format: "HL7"
+    customerStatus: "testing"
+    schemaName: "metadata/fhir_transforms/senders/default-sender-transform"
+    processingType: "sync"
+    allowDuplicates: true
+    senderType: null
+    primarySubmissionMethod: null
+    topic: "elr-elims"
+    version: null
+    createdBy: null
+    createdAt: null
+  receivers: []
```

Comment:
Totally optional, but we can keep things a little bit more concise by getting rid of some of the `null` and `[]` entries here.

Reply:
good call here!